Src/DependencyCollector/NetCore.Tests/DependencyTrackingTelemetryModuleTestNetCore.cs

```diff
@@ -170,29 +170,28 @@ public async Task TestDependencyCollectionWithParentActivity()
 
                 parent.Stop();
 
-                this.ValidateTelemetryForDiagnosticSource(this.sentTelemetry.Single(), url, request, true, "200", false, parent);
+                this.ValidateTelemetryForDiagnosticSource(this.sentTelemetry.Single(), url, request, true, "200", false, true, parent);
 
                 Assert.AreEqual("k=v", request.Headers.GetValues(RequestResponseHeaders.CorrelationContextHeader).Single());
             }
         }
 
         /// <summary>
-        /// Tests dependency collection when request procession causes exception (DNS issue).
-        /// On .netcore1.1 and before, such dependencies are ot collected
-        /// On .netcore2.0 they are collected, but there is no build infra to support it (https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/572)
-        /// TODO: add tests for 2.0
+        /// Tests dependency collection when request procession causes exception (DNS issue).              
         /// </summary>
         [TestMethod]
-        [Timeout(10000)]
+        
         public async Task TestDependencyCollectionDnsIssue()
         {
             using (var module = new DependencyTrackingTelemetryModule())
             {
                 module.Initialize(this.config);
 
-                var request = new HttpRequestMessage(HttpMethod.Get, $"http://{Guid.NewGuid()}");
+                var url = $"http://{Guid.NewGuid()}:5050";
```

Comment:
what is port 5050 ?
would this test still work if we used a non-routable IP address? (ex: 10.0.0.0)

Reply:
5050 was just random!
The goal of this test is to simulate a clientside exception - the easiest way to trigger that is to fail in DNS itself. Its not easy to simulate any other network situation in unit tests.